src/Universum/Unsafe.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE Unsafe #-}
+
+-- | Unsafe functions to work with lists and 'Maybe'.
+-- Sometimes unavoidable but better don't use them. This module
+-- is not even included in default prelude exports.
```

Comment:
Could you also add into documentation that this module is intended to be imported `qualified`? With import example.